### ISSUE TITLE
[Screen Readers - LxP Assist]: Incorrect role as toggle button is defined for 'Register' button.

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4826]
### **PWD Impact:**
As incorrect role is defined for toggle button, it will create confusion for any screen reader dependent users. If screen reader announces incorrect role, users relying on screen reader would be unable to guess the correct functionality of controls present on the page which prevents the visually disabled users to perform their essential task.

### **Repro Steps:**
Step 1: Open the above URL in New Edge /Chrome browsers with valid credentials.
Step 2: 'Enterprise Skills Initiative' page will appear.> Navigate to the 'Click to open Copilot' button and click on it. > 'LXP Assist' flyout will appear->Navigate to Register control using tab key.
Step 3: Verify whether Incorrect role as toggle button is defined for 'Register' button or not.

### **Actual Result:**
Incorrect role as toggle button is defined for 'Register' button.
Screen reader announces: 'Register' toggle button not pressed.

### **Expected Result:** 
Correct role as button should be defined for 'Register' button.
Screen reader should announce: 'Register' button.

### **MAS Reference:**
[MAS 4.1.2 – Name, Role, Value](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7bB661D774-60D0-4277-A121-73F986EBEE76%7d&file=MAS%204.1.2%20%E2%80%93%20Name%2c%20Role%2c%20Value.docx&wdLOR=cE56E09CE-4478-4D20-B785-78D1B90569C1&action=default&mobileredirect=true)

### **WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
### **Test Environment:**
OS: Windows 11 22H2 (OS Build 22621.1848)
Browsers: Microsoft Edge Version (114.0.1823.58), Chrome (114.0.5735.199)
Screen Readers: NVDA (2023.1), JAWS (2023.2303.144)
URL: [Enterprise Skills Initiative INT: Login (microsoft.com)](https://ui.int.esi.microsoft.com/landing)

**Comment by pgopinadhu** [comment]
![image](https://github.com/microsoft/BotFramework-WebChat/assets/120019572/9aa81ae0-5478-41ba-9eb9-06f2db719506)

**Comment by pgopinadhu** [comment]
#HCL; #Accessibility; #Benchmark; #A11yMAS; #A11ySev2; #MAS4.1.2; #Win11-Edge(Chromium); #AILimited; #FeatureBotframeworkwebchat-Apr23; #WWL:Bug1654479; #WWL; #Win11-Chrome;

**Comment by compulim** [comment]
This is related to Adaptive Cards, not Web Chat.

![image](https://github.com/microsoft/BotFramework-WebChat/assets/1622400/52997c70-261c-4e25-8543-7a46c7f8d7fa)

Also, Adaptive Cards is expected to be repetitively submitted. If this is not in the case, please use suggested actions instead.

Closing as this is not an issue on Web Chat. It's either Adaptive Cards or bot author need to redo the card.
